docs/basic-docs/atomicdex-api-20-dev/enable_z_coin_status.md

```diff
@@ -0,0 +1,213 @@
+## task::enable_z_coin::status
+
+After initiating z coin enabling, you can use the `task_id` to check progress.
+
+
+#### Arguments
+
+| Parameter          | Type    | Description                                                                               |
+| ------------------ | ------- | ----------------------------------------------------------------------------------------- |
+| task_id            | integer | The identifying number returned when initiating the initialisation process.               |
+| forget_if_finished | boolean | If `false`, will return final response for completed tasks. Optional, defaults to `true`  |
+
+
+#### Response
+
+| Structure              | Type              | Description                                                                                                           |
+| ---------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------------- |
+| status                 | string            | A short indication of how the enabling is progressing.                                                                |
+| details                | object            | Depending on the state of enabling progress, this will contain different information as shown in the responses below. |
+
+
+#### :pushpin: Examples
+
+#### Command
+
+```bash
+#!/bin/bash
+source userpass
+curl --url "http://127.0.0.1:7783" --data "
+{
+    \"userpass\": \"${userpass}\",
+    \"method\": \"task::enable_z_coin::status\",
+    \"mmrpc\": \"2.0\",
+    \"params\": {
+        \"task_id\": $1
```

Comment:
"task_id": $1,
Missing comma here and extra trailing one below

Reply:
thanks,fixed.